rollup.config.js

```diff
@@ -0,0 +1,51 @@
+import vue from 'rollup-plugin-vue'
+import { terser } from 'rollup-plugin-terser'
+import replace from '@rollup/plugin-replace'
+import filesize from 'rollup-plugin-filesize'
+import resolve from 'rollup-plugin-node-resolve'
+import commonjs from '@rollup/plugin-commonjs'
+import babel from 'rollup-plugin-babel'
+import json from '@rollup/plugin-json'
+import builtins from 'rollup-plugin-node-builtins'
+import globals from 'rollup-plugin-node-globals'
+
+const production = !process.env.ROLLUP_WATCH
+
+// We can't really do much about circular dependencies in node_modules
```

Comment:
Is this really needed?

Reply:
Well it's just about not seeing annoying errors which we can't do anything about 🤷‍♂